net.sourceforge.quakespasm.Quakespasm.appdata.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">net.sourceforge.quakespasm.Quakespasm</id>
+  <name>Quakespasm</name>
+  <developer_name>Quakespasm contributors</developer_name>
+  <summary>A modern, *nix-friendly source port of the original Quake engine</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-only</project_license>
+  <url type="homepage">https://quakespasm.sourceforge.net/</url>
+  <description>
+    <p>Quakespasm is a *Nix friendly Quake Engine based on the SDL port of the popular FitzQuake.
+      It includes some new features, important fixes, and aims for portability and 64 bit correctness.</p>
+    <p>Features:
```

Comment:
Is there a way to preview the description as it would render on Flathub? I've taken my best guess at how the structure wants to be for the list, but I can't 100% confirm it's right.